include/librealsense2/hpp/rs_device.hpp

```diff
@@ -837,7 +837,7 @@ namespace rs2
 
         void on_calibration_change( rs2_calibration_status status ) noexcept override
         {
-            _callback( status );
+            _callback(status);
```

Comment:
Unneeded change. We try not to push changes that are only style, as it is added to the history of the file and make  it more difficult to track a change source (blame).
Also our style does include spaces in the parentheses. You can use attached .clang-format file for ready style definitions.
[clang-format.zip](https://github.com/IntelRealSense/librealsense/files/13626639/clang-format.zip)

Reply:
The spaces were removed. I have one .clang-format file but it not works for me. I customized VS according to Nir's mail.